save-deploy/reverse-proxy.conf

```diff
@@ -11,7 +11,7 @@ server {
     error_log /var/log/nginx/reverse-error.log;
 
     location / {
-        proxy_pass http://127.0.0.1:5000;
+        proxy_pass http://127.0.0.1:5300;
```

Comment:
I won't remove nginx config completely just yet, need to figure out if gateway can replace it completely